cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -31,5 +33,43 @@ function up() {
 EOF
 
     setup_kind
+
+    # remove the rancher.io kind default storageClass
+    _kubectl delete sc standard
+
+    nodes=$(_kubectl get nodes -o=custom-columns=:.metadata.name | awk NF)
+    for node in $nodes; do
+        # Create local-volume directories, which, on other providers, are pre-provisioned.
+        # For more info, check https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/STORAGE.md
+        for i in {1..10}; do
+            mount_disk $node $i
+        done
+        $DOCKER exec $node bash -c "chmod -R 777 /var/local/kubevirt-storage/local-volume"
+
+        # Create a unique UUID file reference file for each node, that will be mounted in order to support
+        # Migration in Kind providers.
+        $DOCKER exec $node bash -c 'cat /proc/sys/kernel/random/uuid > /kind/product_uuid'
+    done
+
+    # create the `local` storage class - which functional tests assume to exist
+    _kubectl apply -f $KIND_MANIFESTS_DIR/local-volume.yaml
+
+    # Since Kind provider uses containers as nodes, the UUID on all of them will be the same,
+    # Migration by libvirt would be blocked, because migrate between the same UUID is forbidden.
+    # Enable PodPreset so we can use it in order to mount a fake UUID for each launcher pod.
+    $DOCKER exec "${CLUSTER_NAME}-control-plane" bash -c 'sed -i \
```

Comment:
we dont support yet migration on SRIOV, so its a dead code for now, i think we shouldnt add it when we dont use it,
it was only for migration

see please
https://github.com/kubevirt/kubevirtci/pull/336

Reply:
Generally I agree, its wont be in use for now until we will need to support migration with SRIOV devices, I will remove it 
Thanks for the heads up.

Note on the tests, this change enables PodPreset feature on the cluster like other features you would enable
we dont need to test it here, you should trust k8s to do his job (if it doesnt well we found a bug)
The code that will make use of the PodPreset (probably kubevirt tests) should also test that it actually took affect.

Reply:
Please see
"Moreover this code isnt tested, you need migration to test it end to end, or at least test that it does the job it should e2e (including kubevirt)"

It should be checked e2e this what i meant, e2e in our context is including kubevirt, even on other PR, but as a step before kubevirtci code is merged.
One can even say i checked locally kubevirtci, we should test, either manually or automatically.
There is enormous benefit to tests things e2e.